index.html

```diff
@@ -6,8 +6,30 @@
     <link rel="stylesheet" type="text/css" href="css/index.css">
   </head>
   <body>
-      <div>
-        <img src="images/homepage-background-2.jpg"/>
```

Comment:
This line has been removed, and you should had to make a pull before you make your commits and your modifications